ozy/config.py

```diff
@@ -38,6 +42,8 @@ def apply_overrides(source, destination):
 
 
 def resolve(config, templates):
+    if 'version' not in config:
```

Comment:
I wouldn't have put this here. `resolve` is not necessarily specific to app config: it could work for any configuration. This app-specific check could be in `App` somewhere (which it is, I think?) Was there a case where you needed this?

Reply:
line 27 of app.py checks to make sure version is there, so I (erroneously, it sounds like?) thought it was a required field that had been missed. I will remove this check (and the subsequent exception that gets thrown if its not there.)

Reply:
ah. I remember now. this was a cheeky way to test line 50 of config.py, specifically, this todo:

` # TODO had these the wrong way round to start with. make a test`

with the version check turned off, this code in test_config won't fire:

```
    # try intentionally switching them
    with pytest.raises(OzyError):
        resolve(config=templates, templates=config)
```